bin/handlemail

```diff
@@ -207,23 +208,29 @@ sub handle_non_bounce_to_null_address {
 
     $text = FYR::Queue::wrap(FYR::Queue::EMAIL_COLUMNS, $text);
 
+    my $html = FYR::Queue::build_html_email($template, {}, $vv);
+
     my $from_name = FYR::Queue::email_sender_name();
     my $from_email = FYR::Queue::do_not_reply_sender();
 
     # We generate this as a bounce.
-    my $mail = MIME::Entity->build(
-                    Sender => '<>',
-                    From => mySociety::Email::format_email_address($from_name, $from_email),
-                    To => $data{return_path},
-                    Subject => 'Automatic reply to your message to WriteToThem',
-                    Date => strftime('%a, %e %b %Y %H:%M:%S %z', localtime(FYR::DB::Time())),
-                    Type => 'text/plain; charset="utf-8"',
-                    Encoding => 'quoted-printable',
-                    Data => FYR::Queue::as_utf8_octets($text)
-                );
+    my $mail = Email::MIME->create(
+        header_str => [
+            Sender => '<>',
+            From => mySociety::Email::format_email_address($from_name, $from_email),
+            To => $data{return_path},
+            Subject => "Automatic reply to your message to WriteToThem",
+            Date => strftime('%a, %e %b %Y %H:%M:%S %z', localtime(FYR::DB::Time())),
+        ],
+        parts => [ $text, $html ],
```

Comment:
I think `$text` should be turned into an Email::MIME object first, otherwise I think if it contains any non-ASCII it will be encoded in base64 as an application/x-binary part.